src/components/main/MainScreen.js

```diff
@@ -10,8 +10,25 @@ import Item from '../common/Item';
 import { hooks } from '../../config/queryClient';
 import HeaderRightContent from './HeaderRightContent';
 import ItemHeader from '../common/ItemHeader';
+import PinnedItems from '../common/PinnedItems';
 
 const MainScreen = () => {
+ const getParentsIdsFromPath = (path) => {
```

Comment:
I'm not sure but I think this function exists in `utils`. Otherwise it could be a good place to put it.